build.gradle.kts

```diff
@@ -52,7 +52,7 @@ dependencies {
     implementation("io.micrometer:micrometer-registry-prometheus")
 
     implementation("io.github.microutils:kotlin-logging:1.8.3")
-    implementation("net.devh:grpc-spring-boot-starter:2.9.0.RELEASE")
+    implementation("net.devh:grpc-spring-boot-starter:2.10.1.RELEASE")
```

Comment:
Don't update dependency without checking it's dependencies. For example, when updating this, you should also update the following:  
    grpc-java v1.31.1
    protobuf v3.12.4
    spring-boot v2.3.3

Check release notes:
https://github.com/yidongnan/grpc-spring-boot-starter/releases/tag/v2.10.1.RELEASE